Cargo.toml

```diff
@@ -11,6 +11,9 @@ thiserror = "1.0.30"
 smallvec = "1.8.0"
 pyo3 = {version = "0.17.1", features = ["extension-module"], optional = true}
 quickcheck = {version = "1.0.3", optional = true}
+serde = {version = "1.0", features = ["derive"], optional = true}
+serde_json = {version = "1", optional = true}
```

Comment:
As far as I can tell, `serde_json` is only used in tests. Would it be possible to move that to a dev dependency?